cmake/extension.cmake

```diff
@@ -247,6 +247,17 @@ endif()
 
 if(${HTTPPROXY} STREQUAL true)
   client_sdk_compile_definitions(-DHTTPPROXY)
+  if(${PROXY_DISCOVERY} STREQUAL true)
```

Comment:
Could you please update commit message for PROXY_DISCOVERY change as well?

Reply:
@trbehera please re-review.  fixed the commit.